nss-day-cohort-48/rare-client-polite-pirates
# Mrw comments

#### Changes Made

1. Added file `commentList.js, commentForm.js, commentList.css` to `comments` directory.

   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout mrw-comments
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   >go to a post and click comments. their should be all of your comments.
4. View code file.
   > Confirm file modifications are present as indicated above.
